tests/puppeteer/campaigns/data/faker/tax.js

```diff
@@ -0,0 +1,10 @@
+const faker = require('faker');
+
+module.exports = class Tax {
+  constructor(taxToCreate = {}) {
+    this.rate = taxToCreate.rate || faker.random.number({min: 1, max: 100}).toString();
```

Comment:
Maybe limit this to 40 :thinking: ? Having a 100% tax is kinda harsh :joy: 
```suggestion
    this.rate = taxToCreate.rate || faker.random.number({min: 1, max: 40}).toString();
```

Reply:
done, thanks